### ISSUE TITLE
Update - Check Domoticz server for updates on startup

### ISSUE THREAD

**Issue by RoadXY** [galadril/DomoticzForAndroid#33]
Don't know if it'll be a nice feature though.

But there is a JSON call for that:
/json.htm?type=command&param=checkforupdate&forced=true

Any thoughts on this?

**Comment by galadril** [comment]
I don't really get why you would like to update Domoticz from your mobile...?
But here are the API calls:

Is there an update:
/json.htm?type=command&param=checkforupdate&forced=true

Trigger update:
/json.htm?type=command&param=downloadupdate

Check if update ready:
/json.htm?type=command&param=downloadready

**Comment by RoadXY** [comment]
I agree, only notify the user there is a update available, nothing more.
And only once for that specific update, not every time for that same specific update.

I think it will add positive experience for the user.

**Comment by galadril** [comment]
Added in v0.1.137
I've added a small Toast notification when an update is available, and it's visible under the settings!

![screenshot_20151124-083501](https://cloud.githubusercontent.com/assets/14561640/11360610/7311d776-9286-11e5-9235-09e8e262ad6e.png)
![screenshot_20151124-083412](https://cloud.githubusercontent.com/assets/14561640/11360611/731ad2ae-9286-11e5-8e16-8588e95b53e1.png)

**Comment by RoadXY** [comment]
Nice!
This is something I was thinking about.

A little suggestion: could the toast message be changed to "Domoticz Server update available: 1234"
Maybe?
I think some users will otherwise think there is an Domotica Android app update available.

**Comment by galadril** [comment]
I'll change it, its in the strings.xml

**Comment by RoadXY** [comment]
I can do it for you, tonight, if you like?

**Comment by galadril** [comment]
Its already done
